index.html

```diff
@@ -6,15 +6,124 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lemonada">
+    
     <!-- Add a link to your custom CSS here -->
-
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
-    <!-- Add your markup here -->
+<header>
+    <div class="row head-logo">
+        <h1 class="logo">M&S MOTORS</h1> 
+    </div>
+    <ul class="row list ">
+        <li class="sm-col-12 md-col-4 lg-col-4">
+            <a href="#projects">projects.</a>
+        </li>
+        <li class="sm-col-12 md-col-4 lg-col-4">
+            <a href="#about">about.</a>
+        </li>
+        <li class="sm-col-12 md-col-4 lg-col-4">
+            <a href="#contact">contact.</a>
+        </li>
+    </ul>
+    
+</header>
+     
+<main>
+<section class="header-hero">
+    
+</section>
+<section class="container hero-content">
+<h3>projects.</h3>
 
-</body>
+<div class="row">
+    <div class="sm-col-12 md-col-12 lg-col-4">
```

Comment:
You should start your column classes with `col-...` not `sm-col-...` as `sm` starts from 576px, so small mobile phones (like iPhone 4/5) are not included.